keyboards/crkbd/rev1/matrix.c

```diff
@@ -200,19 +205,9 @@ uint8_t matrix_scan(void)
         matrix_master_scan();
     }else{
         matrix_slave_scan();
-
-//        if(serial_slave_DATA_CORRUPT()){
-//          TXLED0;
-          int offset = (isLeftHand) ? ROWS_PER_HAND : 0;
-
-          for (int i = 0; i < ROWS_PER_HAND; ++i) {
-              matrix[offset+i] = serial_master_buffer[i];
-          }
-
-//        }else{
-//          TXLED1;
-//        }
-
+        int offset = (isLeftHand) ? ROWS_PER_HAND : 0;
+        memcpy(&matrix[offset],
+               (void *)serial_master_buffer, sizeof(serial_master_buffer));
```

Comment:
OIC. sizeof(serial_slave_buffer) as same as  ROWS_PER_HAND.

However, if the size to be copied to memory exceeds the size secured by the variable, it causes a bug, so I think that this part is acceptable as it is.

Therefore, I think it would be better to delete ROW_PER_HAND and use  SERIAL_SLAVE_BUFFER_LENGTH or SERIAL_MASTER_BUFFER_LENGTH.

What do you think?